### PR TITLE
Add Summary of Network Information

### PR DESCRIPTION
**Pull request by krippz**
This switch seems to be undocumented in the man page, it gives a summary of all network interfaces. It shows info like nic bsd name, current ip address for ipv4 and ipv6

### CODE REVIEW

**Comment by krippz** [comment]
I'm sorry but I don't understand the error I'm getting on this build. please advise

**Comment by rthille** [comment]
Given the `403` in the Issues just before the link, maybe the checker got a 403 error from Apple when trying to retrieve the URL?  I couldn't reproduce but maybe Apple is blocking based on user agent or source IP?

**Comment by rcritz** [comment]
The CI failure has nothing to do with this PR. As @rthille says, it's related to an Apple link in the section on the `srm` command.

**Comment by herrbischoff** [comment]
This is fixed now.

**Comment by herrbischoff** [comment]
Merging. @krippz: please note my changes, also as a guide to future additions. `--nwi` is actually an undocumented flag and therefore interesting.

File: README.md [hunk 1]
```diff
@@ -1337,8 +1337,9 @@ Alternative that works on all networks.
 ```bash
 curl -s https://api.ipify.org && echo
 ```
-#### Show Summary of Network Information
-This displays all your network interface information, both for IPv4 and IPv6.
+
+#### Show Network Interface Information
+Undocumented flag of the `scutil` command.
```
Comment (dominiklohmann) [inline README.md]: This is a documented flag. From `scutil --help`:

```
or: scutil --nwi
    show network information
```

Comment (herrbischoff) [reply README.md]: Interesting. It’s not documented in the man file, which I usually consider documentation. Thanks for the heads up.

**Pull request merged**